yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/services/StatisticsServiceImpl.java

```diff
@@ -85,11 +83,13 @@ public GlobalStatisticsEntity getGlobalStatistics(StatisticsRequestEntity reques
         double declarationCount = Double.parseDouble(result[0].toString());
         double remoteCount = Double.parseDouble(result[1].toString());
         double onsiteCount = Double.parseDouble(result[2].toString());
+        double absenceCount = Double.parseDouble(result[3].toString());
```

Comment:
beware of the double in Java. Use BigDecimal instead to avoir floating point bug